### ISSUE TITLE
Grouping /Ungrouping 

### ISSUE THREAD

**Issue by michshat** [JanKallman/EPPlus#535]
Thanks everyone for supporting this tool.

There is seem to be support for grouping /ungrouping, but can't seem to figure it out how use it. I've found few example that refer to `OutlineLevel` as a mean to group (one example https://stackoverflow.com/a/32763961/297827) However i'm looking into something similar that is described here https://docs.microsoft.com/en-us/visualstudio/vsto/how-to-programmatically-group-rows-in-a-worksheet?view=vs-2019

I thought the idea would be to get the range of rows and get theme `.Group` or `Ungroup`

Any idea?

**Comment by panjevic** [comment]
@michshat Did you manage to understand how to do ungroup on already existing groups?

**Comment by michshat** [comment]
@panjevic Yes, I did achieve it via OutlineLevel, apparently for it to work I needed to have a row in between the groups of rows (the group header row sort to speak). I was able to create two sublevels using this technique. All is good - Thank you!
